packages/peregrine/lib/PeregrineContextProvider/__tests__/__snapshots__/peregrineContextProvider.spec.js.snap

```diff
@@ -1,17 +1,17 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`render wraps children with all peregrine context providers 1`] = `
-<Connect(ErrorContextProvider)>
-  <Connect(AppContextProvider)>
-    <Connect(UserContextProvider)>
-      <Connect(CatalogContextProvider)>
-        <Connect(CartContextProvider)>
-          <Connect(CheckoutContextProvider)>
+<Memo(ConnectFunction)>
+  <Memo(ConnectFunction)>
+    <Memo(ConnectFunction)>
+      <Memo(ConnectFunction)>
+        <Memo(ConnectFunction)>
+          <Memo(ConnectFunction)>
```

Comment:
Feels nitpicky but snapshots like this aren't very useful, lol. Maybe if we put the number in the test name? "...wraps children with all six context providers" ?

Reply:
Yea it's not helpful. The only reason this is changing is because feature branch was broken but your complaint, at it's core, is very relevant. We should delete this or assert on the number of contexts being provided or something...